pkg/js/validator/validate-dsl.ts

```diff
@@ -117,6 +121,19 @@ const relationIsSingle = (currentRelation: Userset): boolean => {
     );
 };
 
+const getRelationDefName = (userset: Userset): string | undefined => {
```

Comment:
What is this function doing? it is returning the string representation of the partial relation definition? Why is it needed?

Reply:
Yeah, just a small helper function to reduce the amount of if-else cases[ in the following block.](https://github.com/openfga/language/pull/36/files/851d7e555e15c1bb7df8f1e03fc52183c4a9063e#diff-e41962816c915ede1d4ccf0db08eee9fedfd28d62a787698ce1d8a94188f3fe2R357)